.github/ISSUE_TEMPLATE/redirect-request.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Redirect Request
+about: Submit a request to implement a redirect for a URL
+title: Redirect Request
+labels: content-ia-team, ia
+assignees: mnorthuis
+
+---
+
+### Instructions
+- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
+- Please add your team's label to this issue and complete the information below.
+- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple tickets to track differnt steps.
+- It is the responsibility of the requesting team to notify VA stakeholders of pending redirect.  
+
```

Comment:
We should minimize the "Instructions" area, assuming few people will read it completely through. These all do seem important, but maybe we can cut excess words to make it more easily skim-able. Proposed changes: 

- Bullet 1, cut the second sentence? Not sure?
- Cut bullet 2 (and just add to steps below)
- Bullet 3, use "issue" or "ticket" consistently. Typo in "different"
- Bullet 4 specify "you" — "It is YOUR responsibility, as the team requesting the redirect, to notify VA and other stakeholders of pending redirect."